server/controllers/followersController.js

```diff
@@ -36,15 +36,18 @@ exports.getFollowersUnfollowers = function (req, res) {
 
 }
 
-// GET /user/:userId
+// GET /user/:userId/followers
 exports.getFollowers = function (req, res) {
     console.log('Get followers called');
     const userId = req.params.userId;
     if (!userId)
         return res.status(400).send('Bad Request');
     return getFollowers(userId)
         .then(followers => {
-            res.status(200).json(followers);
+            res.status(200).json({followers: followers});
+        })
+        .catch(err => {
+            res.status(404).json({followers: [], message: err});
         });
```

Comment:
I am not sure about error handling. I did this to be able to show "There is no followers" in the client.